.gitmodules

```diff
@@ -1,7 +1,6 @@
 [submodule "3rdparty/tvm"]
 	path = 3rdparty/tvm
 	url = https://github.com/neo-ai/tvm.git
-        branch = stable
```

Comment:
do we need to change this?

Reply:
this change is made by https://github.com/neo-ai/neo-ai-dlr/pull/189 that has been merged in the master branch.

Reply:
> Shouldn't we revert the changes from tvm?

The changes from tvm are already up to date with master.

Reply:
I see. You need to rebase your change again the master branch of dlr because you are currently incorporating other commits.

Reply:
rebased.